app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -54,4 +59,13 @@ class MainActivity : AppCompatActivity() {
                 else -> View.GONE
         }
     }
+
+    private fun snackbar() {
```

Comment:
It would be better if the function name was more meaningful

Reply:
is 'isSnackbar' fine? 😅